spec/features/admin/variants_spec.rb

```diff
@@ -118,4 +118,29 @@
     v.reload
     v.deleted_at.should_not be_nil
   end
+
+  scenario "editing display name for a variant", js:true do
+    p = create(:simple_product)
+    v = p.variants.first
+
+    # When I view the variant
+    login_to_admin_section
+    visit spree.admin_product_variants_path p
+    page.find('table.index .icon-edit').click
+
+    # It should allow the display name to be changed
+    page.should have_field "variant_display_name"
+    page.should have_field "variant_display_as"
+
```

Comment:
Great that you added a test! :slightly_smiling_face:

It's okay to leave this as is, but for future pull requests, please use the newer `expect` syntax. https://github.com/rubocop-hq/rspec-style-guide#use-expect